### ISSUE TITLE
Release applications crash due to class resolution failure

### ISSUE THREAD

**Issue by tytydraco** [Swordfish90/LibretroDroid#21]
Calling getVariables() on a GLRetroView can cause a crash on release versions due to proguard stripping the class away. The issue is not present on debug builds.

https://del.dog/lirfatoora.txt

In my limited experience with proguard, I think we need to add proguard rules to keep the class available.

**Comment by Swordfish90** [comment]
Yes, definitely. A valid consumerProguardFile should be provided. As a workaround, you can add this to your app proguard file.

-keep class com.swordfish.libretrodroid.** { *; }

**Comment by tytydraco** [comment]
Bump!
